core/tests/test_utils.py

```diff
@@ -143,16 +148,39 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Returns a string for unicode compatibility test
+
+        Args:
+            suffix: string . Test string appended to unicode string
```

Comment:
There is no need of a space between "string" and ".", right ? Please format it like:
`suffix: string. Test string ...`.
Also one doubt, isn't suffix just a string to be appended to unicode test string ?